omnibus/files/private-chef-cookbooks/private-chef/recipes/bookshelf_database.rb

```diff
@@ -0,0 +1,37 @@
+bookshelf_attrs = node['private_chef']['bookshelf']
+postgres_attrs = node['private_chef']['postgresql']
+
+# create users
+private_chef_pg_user bookshelf_attrs['sql_user'] do
+  password bookshelf_attrs['sql_password']
+  superuser false
+end
+
+private_chef_pg_user bookshelf_attrs['sql_ro_user'] do
+  password bookshelf_attrs['sql_ro_password']
+  superuser false
+end
+
+private_chef_pg_database 'bookshelf' do
+  owner bookshelf_attrs['sql_user']
+end
+
+private_chef_pg_user_table_access bookshelf_attrs['sql_user'] do
+  database 'bookshelf'
+  schema 'public'
+  access_profile :write
+end
+
+private_chef_pg_user_table_access bookshelf_attrs['sql_ro_user'] do
+  database 'bookshelf'
+  schema 'public'
+  access_profile :read
+end
+
+private_chef_pg_sqitch "/opt/opscode/embedded/service/bookshelf/schema" do
+  hostname postgres_attrs['vip']
+  port     postgres_attrs['port']
+  username  postgres_attrs['db_superuser']
+  password  postgres_attrs['db_superuser_password']
+  database "bookshelf"
+end
```

Comment:
I think we decided that we want to make all oc_erchef, bifrost, and bookshelf all look the same?  For clarity, it might be best to do that in a follow up commit shortly after this is merged.